src/components/Slider/StandardSlider/StandardSlider.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import styles from '../Slider.module.scss';
+import classNames from 'classnames/bind';
+const cx = classNames.bind(styles);
+
+import {
+  Root as SliderRoot,
+  Track as SliderTrack,
+  Range as SliderRange,
+  Thumb as SliderThumb
+} from '@radix-ui/react-slider';
+
+export interface SliderProps {
+  step?: number;
+  min?: number;
+  max?: number;
+  value?: number;
+  className?: string;
+  onValueChange?: (value: number) => void;
```

Comment:
Noticed that this prop isn't being used anywhere - it should be passed to `onValueChange` on `SliderRoot`.